api/src/main/resources/workbench.yaml

```diff
@@ -935,6 +935,23 @@ paths:
             $ref: "#/definitions/CriteriaListResponse"
 
   # Cohort Review  #######################################################################
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/participants/filters:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+    get:
+      tags:
+        - cohortReview
+      description: Will return a list all values for gender, race and ethnicity.
+      operationId: "getParticipantFilters"
```

Comment:
I'm not super in love with calling these "Filters" - that's what they'll be _used in_, not what they _are_, but a better name doesn't occur to me atm.  I'd say this is good enough for now and probably low hanging fruit for a rename / refactor down the road.

Reply:
getParticipantDemographics?